### ISSUE TITLE
Type signature for Enzyme's contains method doesn't match documentation

### ISSUE THREAD

**Issue by scharles-resilient** [DefinitelyTyped/DefinitelyTyped#52484]
Hi @ljharb, 
The current type [signature](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/enzyme/index.d.ts) for Enzyme's contains method accepts numbers, strings, ReactElements as well as arrays of ReactElements. 

However, the current [documentation](https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/contains.html) for Enzyme suggests that the method should only accept a ReactElement or an array of such elements. 
 
I suggest the type signature is updated to no longer allow strings and numbers.

**Comment by ljharb** [comment]
Indeed, that's a bug in the types. They should only accept `ReactElement | ReactElement[]`.

**Comment by ljharb** [comment]
Looks like this is actually not a bug for the shallow wrapper's `contains`, at least - the implementation definitely accepts a string or a number.
